Dockerfile

```diff
@@ -5,6 +5,7 @@ ARG ARCH=x86_64
 ENV TERM=xterm \
     APPLIANCE=true \
     RAILS_USE_MEMORY_STORE=true
+RUN dnf -y update
```

Comment:
Would moving to `ubi:8.4` also fix this issue?

Reply:
Just updated the changes with `ubi:8.4` which also needs the dnf update.